src/test/resources/archives/case_claim_with_multi_select/suite.xml

```diff
@@ -232,7 +232,7 @@
     <command id="m1-f1"/>
   </menu>
   <remote-request>
-    <post url="http://localhost:8000/a/test/phone/claim-case/" relevant="true()">
+    <post url="http://localhost:8000/a/test/phone/claim-case/" relevant="$case_id != ''">
```

Comment:
Does `$case_id` refer to the value of the data element? Where is that set in the eval context?

Reply:
Ah: https://github.com/dimagi/commcare-core/pull/1097